<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferCheckDialog</class>
 <widget class="QDialog" name="TransferCheckDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>316</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Transfer Checks</string>
  </property>
  <widget class="QPushButton" name="writeCheckButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Write Check</string>
   </property>
  </widget>
  <widget class="Q3ListBox" name="chargeSummaryListBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>440</width>
     <height>250</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Vera Sans Mono</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="selectionMode">
    <enum>Q3ListBox::Multi</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="colorCombo">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>0-69</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>70-79</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>80-89</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>90-99</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>All</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="textLabel2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Color Group</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ListBox</class>
   <extends>Q3Frame</extends>
   <header>q3listbox.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="global">vector</include>
  <include location="local">dao/chargedao.h</include>
  <include location="local">dao/checkdao.h</include>
  <include location="local">dao/printerdao.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
